### PR TITLE
Added README.md, added 2 exclamation marks, removed semicolons

### PR DESCRIPTION
**Pull request by temeje**
- [x] Added a Readme. 
- [x] Added 2 exclamation marks because I thought a bit more emphasis was necessary.
- [x] Removed semicolons, because Prettier decided to do that

### CODE REVIEW

**Review by keywordnew — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Cambie Collective Node Server
+
+Simple Node/Express app which listens on port 3000 and returns 'Hello World!!!'
+
```
Comment (daffl) [inline README.md]: I think we'll need `npm install` as well 🚀

**Review by daffl — CHANGES_REQUESTED** [review]

**Review by preetdeol — COMMENTED** [review]
hi

**Review by atifali — COMMENTED** [review]
Maybe include 'npm install' step before npm start?

File: README.md (hunk 1 above)
Comment (temeje) [reply README.md]: I agree. Thanks!

Comment (keywordnew) [reply README.md]: Maybe you can use this :)

```suggestion
First run `npm install` to install the dependencies!
```

**Comment by keywordnew** [comment]
I think all the comments are resolved. Merging!

**Pull request merged**